Templates/scripts/signin.js

```diff
@@ -0,0 +1,10 @@
+function unhide() {
+    var x = document.getElementById("login-form");
```

Comment:
would you mind finding a good name for this variable?? try to avoid using x,y,z, ...
a variable name should be more descriptive about what it holds